pkg/trace/agent/agent.go

```diff
@@ -308,18 +312,12 @@ func (a *Agent) convertStats(in pb.ClientStatsPayload, lang, tracerVersion strin
 		}
 		in.Stats[i].Stats = group.Stats[:n]
 	}
-	return pb.StatsPayload{
-		Stats:          []pb.ClientStatsPayload{in},
-		AgentEnv:       a.conf.DefaultEnv,
-		AgentHostname:  a.conf.Hostname,
-		AgentVersion:   info.Version,
-		ClientComputed: true,
-	}
+	return in
 }
 
 // ProcessStats processes incoming client stats in from the given tracer.
 func (a *Agent) ProcessStats(in pb.ClientStatsPayload, lang, tracerVersion string) {
```

Comment:
```suggestion
func (a *Agent) AggregateStats(in pb.ClientStatsPayload, lang, tracerVersion string) {
```

Reply:
This name bubbles up to the api.go interfaces. It matches the one used for traces in agent.go. While renaming the files in api.go it felt a bit weird. Should I do this rename, if yes I'll do it in next PR?